custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -67,6 +69,19 @@ def __init__(self, model_dir, target_type=None):
         self._model_dir = model_dir
         self._target_type = target_type
 
+        # New custom task class and instance loaded from custom.py
+        self._custom_task_class = None
+        self._custom_task_class_instance = None
+
+    @property
+    def is_custom_task_class(self):
+        """
+        Returns
+        -------
+        True if the code is using the CustomTask class interface. False if it's using the legacy drum hook interface.
+        """
+        return self._custom_task_class
```

Comment:
doesn't seem very bool to me

Reply:
need to update to `is not None`

Reply:
Fixed. Mind making another pass on this PR